src/qt/guiutil.cpp

```diff
@@ -1013,43 +1015,92 @@ const std::vector<QString> listThemes()
     return vecThemes;
 }
 
-// Open CSS when configured
-QString loadStyleSheet()
+void loadStyleSheet(QWidget* widget, bool fDebugWidget)
 {
-    static std::unique_ptr<QString> stylesheet;
+    AssertLockNotHeld(cs_css);
+    LOCK(cs_css);
 
-    if (stylesheet == nullptr) {
-        stylesheet = std::make_unique<QString>();
+    static std::unique_ptr<QString> stylesheet;
+    static std::set<QWidget*> setWidgets;
+
+    bool fDebugUI = gArgs.GetBoolArg("-debug-ui", false) && isStyleSheetDirectoryCustom();
```

Comment:
The name is confusing imo. Maybe smth like
```suggestion
    bool fReloadStyleSheet = gArgs.GetBoolArg("-debug-ui", false) && isStyleSheetDirectoryCustom();
```
? (with corresponding changes below ofc)

Reply:
What about `fDebugStylesheets` ?

Reply:
`fReloadStyleSheet` conflicts a bit with https://github.com/dashpay/dash/blob/6ae1999d2f36398c3b59071c40f7e9bd56d089e6/src/qt/guiutil.cpp#L1079 imo

Reply:
Hmm.. How about `fDebugCustomStyleSheets` to be even more verbose/precise?

Reply:
works for me :)